frontend/src/Commons/Forms/InputModules/FieldsAndSoil/FieldsAndSoil.tsx

```diff
@@ -66,42 +63,71 @@ const FieldsAndSoilComponent: React.FC<InputModuleProps> = ({ farmDetails, updat
       validationSchema={validationSchema}
       onSubmit={onSubmit}
     >
-      <Form>
-        <StyledFarmInfo>
-          <div id="inputContainer">
-            <CustomField
-              label="Year"
-              id="Year"
-              name="Year"
-              type="number"
-              width="20%"
-            />
-
-            <CustomField
-              label="Farm Name"
-              id="FarmName"
-              name="FarmName"
-              type="text"
-            />
-          </div>
-
-          <StyledSelectContainer>
-            <CustomSelect
-              name="FarmRegion"
-              id="FarmRegion"
-              label="Region"
-              options={options}
-              width="50%"
-            />
-            <Button
-              type="submit"
-              size="sm"
-              disabled={false}
-              text={ComponentText.NEXT}
-            />
-          </StyledSelectContainer>
-        </StyledFarmInfo>
-      </Form>
+      {({ values }) => (
+        <Form>
+          <StyledFarmInfo>
+            <div id="inputContainer">
+              <CustomField
+                label="Field name"
+                id="FieldName"
+                name="FieldName"
+                type="text"
+              />
+              <CustomField
+                label="Area"
+                id="Area"
+                name="Area"
+                type="number"
+                width="20%"
+              />
+            </div>
+            <StyledTextAreaContainer>
+              <CustomTextArea
+                name="Comments"
+                id="Comments"
+                label="Comments (optional)"
+                placeholder="e.g., poor drainage in southwest corner (no need to specify crop here)"
+                width="50%"
+              />
```

Comment:
Awesome! Could be another ticket, but I think we can have an enhancement ticket if needed. Good job!

Reply:
I could have broken it down this component too!